tests/test_smoketest.py

```diff
@@ -52,9 +53,12 @@ def run_for_json(command, **kwargs):
 
 @testmode.integration
 class Smoketest(unittest.TestCase):
-    replicas = "aws", "gcp"
+    replicas = {
+        "aws",
+        "gcp"
+    }
     notification_bucket = os.environ['DSS_S3_BUCKET_TEST']
-    checkout_bucket = os.environ['DSS_S3_CHECKOUT_BUCKET']
+    checkout_bucket = {'aws': os.environ['DSS_S3_CHECKOUT_BUCKET'], 'gcp': os.environ['DSS_GS_CHECKOUT_BUCKET']}
     test_bucket = os.environ['DSS_S3_BUCKET_TEST']
```

Comment:
I think this needs to be parameterized too. 

Might be worth creating a replica to parameterization values dict.